packages/client/components/TeamPrompt/TeamPromptTopBar.tsx

```diff
@@ -134,74 +134,103 @@ const TeamPromptTopBar = (props: Props) => {
   const {handleSubmit, validate, error} = useRenameMeeting(meetingId)
   const isRecurrenceEnabled = meetingSeries && !meetingSeries.cancelledAt
 
+  const onOpenWorkSidebar = () => {
+    commitLocalUpdate(atmosphere, (store) => {
+      const meetingProxy = store.get(meetingId)
+      if (!meetingProxy) return
+      meetingProxy.setValue(null, 'localStageId')
+      meetingProxy.setValue(true, 'showWorkSidebar')
+      meetingProxy.setValue(true, 'isRightDrawerOpen')
+    })
+  }
+
+  const buttons = (
+    <ButtonContainer>
+      <button
+        className='group flex h-max w-max cursor-pointer flex-col items-center px-2 text-sm font-semibold text-sky-500 hover:text-sky-600'
+        onClick={onOpenWorkSidebar}
```

Comment:
-1 can we make this a toggle? e.g. if i click it when the right drawer is open, it closes the drawer? that'll keep the menu trigger consistent with the other ones in the app!

Reply:
Updated - definitely an improvement!